virttest/qemu_monitor.py

```diff
@@ -920,17 +927,15 @@ def cmd(self, cmd, timeout=CMD_TIMEOUT, debug=True, fd=None):
         try:
             # Read any data that might be available
             self._recvall()
-            if fd is not None:
-                if self._passfd is None:
-                    self._passfd = passfd_setup.import_passfd()
-                # If command includes a file descriptor, use passfd module
-                self._passfd.sendfd(self._socket, fd, b"%s\n" % cmd.encode())
-                self._log_lines(cmd)
-            else:
-                # Send command
-                if debug:
-                    LOG.debug("Send command: %s" % cmd)
-                self._send(cmd.encode())
+            # Send command
+            msg = cmd.encode()
+            fds = (
+                [fd.fileno() if not isinstance(fd, int) else fd]
+                if fd is not None else None
```

Comment:
This can be used for inspiration to give a type and a one-sentence description for the file descriptors above.